libmultilabel/linear/metrics.py

```diff
@@ -28,6 +28,10 @@ def update(self, preds: np.ndarray, target: np.ndarray) -> None:
     def compute(self) -> float:
         return self.score / self.num_sample
 
+    def reset(self):
+        self.score = 0
+        self.num_sample = 0
+
```

Comment:
for this fun, should we release a new pypi? If so, something must be done (e.g., version num in setup.py etc)